.github/workflows/playwright.yml

```diff
@@ -0,0 +1,40 @@
+name: Playwright Tests
+
+on:
+  schedule:
+    - cron: "0 1 * * 5"
+  workflow_dispatch:
+jobs:
+  playwright:
+    timeout-minutes: 60
+    runs-on: ubuntu-22.04
+    env:
+      CHROME_EXT_PASSWORD: "1234Neon5678"
+      NETWORK: "devnet"
+      IMAGE: neonlabsorg/neon_tests:b59be97d2daaf2a2d080c24269b2c9edd085986f
+      CONTAINER: neontests-${{ github.run_id }}
+    steps:
+      - name: "Checkout repository"
+        uses: actions/checkout@v3
+      - name: "Pull Docker Test Image"
+        run: docker pull ${{ env.IMAGE }}
+      - name: "Run docker container"
+        run: |
+          docker run -i -d --name=${{ env.CONTAINER }} \
+            -e CHROME_EXT_PASSWORD=${{ env.CHROME_EXT_PASSWORD }} \
+            --entrypoint /bin/bash ${{ env.IMAGE }}
+      - name: "Run Playwright Tests"
+        run: |
+          docker exec -i ${{ env.CONTAINER }} \
+            xvfb-run python3 ./clickfile.py run ui --network=${{ env.NETWORK }} --ui-item=neonpass
+      - name: "Generate allure report"
+        if: always()
+        uses: ./.github/actions/generate-allure-report
+        with:
+          container: ${{ env.CONTAINER }}
+          network: ${{ env.NETWORK }}
+          aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          tests_name: ui
+      - name: "Remove Docker Container"
```

Comment:
we don't need this, because github destroy runner after run

Reply:
Fixed